### PR TITLE
Upgrade axios to 0.18.1

### PR DESCRIPTION
**Pull request by cachedout**
## What does this PR do?

Resolves [security alert](https://github.com/elastic/apm-integration-testing/network/alert/scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock/axios/open) for axios.

## Why is it important?

Actual severity is likely very low but this is an easy fix.
## Related issues
Closes #ISSUE

### CODE REVIEW

File: scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock [hunk 1]
```diff
@@ -18,51 +18,51 @@ arg@^4.1.0:
...
 buffer-from@^1.0.0:
   version "1.1.1"
   resolved "https://registry.yarnpkg.com/buffer-from/-/buffer-from-1.1.1.tgz#32713bc028f75c02fdb710d7c7bcec1f2c6070ef"
   integrity sha512-MQcXEUbCKtEo7bhqEs6560Hyd4XaovZlO/k9V3hjVUF/zwW7KBVdSK4gIt/bzwS9MbR5qob+F5jusZsb0YQK2A==
 
-debug@^3.2.6:
-  version "3.2.6"
-  resolved "https://registry.yarnpkg.com/debug/-/debug-3.2.6.tgz#e83d17de16d8a7efb7717edbe5fb10135eee629b"
-  integrity sha512-mel+jf7nrtEl5Pn1Qx46zARXKDpBbvzezse7p7LqINmdoIk8PYP5SySaxEmYv6TZ0JyEKA1hsCId6DIhgITtWQ==
+debug@=3.1.0:
```
Comment (kuisathaverat) [inline scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock]: It seems a downgrade

File: scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock [hunk 2]
```diff
@@ -18,51 +18,51 @@ arg@^4.1.0:
...
 make-error@^1.1.1:
   version "1.3.5"
   resolved "https://registry.yarnpkg.com/make-error/-/make-error-1.3.5.tgz#efe4e81f6db28cadd605c70f29c831b58ef776c8"
   integrity sha512-c3sIjNUow0+8swNwVpqoH4YCShKNFkMaw6oH1mNS2haDZQqkeZFlHS3dhoeEbKKmJB4vXpJucU6oH75aDYeE9g==
 
-ms@^2.1.1:
-  version "2.1.1"
-  resolved "https://registry.yarnpkg.com/ms/-/ms-2.1.1.tgz#30a5864eb3ebb0a66f2ebe6d727af06a09d86e0a"
-  integrity sha512-tgp+dl5cGk28utYktBsrFqA7HKgrhgPsg6Z/EfhWI4gl1Hwq8B/GmY/0oXZ6nF8hDVesS/FpnYaD/kOWhYQvyg==
+ms@2.0.0:
```
Comment (kuisathaverat) [inline scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock]: It seems a downgrade

File: scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock (hunk 1 above)
Comment (cachedout) [reply scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock]: This is what `yarn upgrade axios` chose to do. If we feel safe to just do a full `yarn upgrade` then we could pull everything up-to-date. What do you think?

Comment (kuisathaverat) [reply scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock]: yep, we could try it

Comment (cachedout) [reply scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock]: @kuisathaverat I've pushed a change from a full `yarn upgrade` and it seems to preserve the downgrades. I'm honestly not sure why this is happening.

Comment (kuisathaverat) [reply scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/yarn.lock]: it is fine, yarn stuff 😃 we could live with that if it works

**Pull request merged**